aks/variables.tf.json

```diff
@@ -53,7 +53,7 @@
         "k8s_version": {
             "type": "string",
             "description": "Kubernetes version to apply to AKS; must be supported in the selected region. (Run `az aks get-versions --location $REGION --output table` for a list of supported options)",
-            "default": "1.15.7"
+            "default": "1.15.11"
```

Comment:
I'd recommend taking out `k8s_version` altogether because it would default to the latest stable AKS release. 

https://www.terraform.io/docs/providers/azurerm/r/kubernetes_cluster.html#kubernetes_version

Perhaps better than having to update this on an ongoing basis given that AKS, unlike GKE, doesn't support specifying a wildcard match for the version.

Reply:
We have a feature planned to request a list of supported versions in each CSP, and present the user an option to select, instead of a string entry... but that's in the next version.

WRT to defaulting to the newest, we can't support that. Azure is now supporting k8s 1.18 , but our supported kubectl is stuck at 1.16, and that is outside the range supported upstream.

Reply:
https://github.com/SUSE-Enceladus/blue-horizon/issues/121

Reply:
I would prefer to pin the k8s_version, makes it easier to discover which kubectl version is needed for which cluster.